src/lapack_wrap/test/CMakeLists.txt

```diff
@@ -10,17 +10,14 @@ project( lapack_wrap_test CXX )
 #--------------------------------------------------------------------------------------------------#
 # Source files
 #--------------------------------------------------------------------------------------------------#
-
 file( GLOB test_sources *.cc )
 
 #--------------------------------------------------------------------------------------------------#
 # Build Unit tests
 #--------------------------------------------------------------------------------------------------#
-
 add_scalar_tests(
    SOURCES "${test_sources}"
-   DEPS    "Lib_dsxx;lapack;blas" )
-
+   DEPS    "Lib_dsxx;LAPACK::LAPACK" )
```

Comment:
@KineticTheory Should there also be a `BLAS::BLAS` here or is that already captured by `LAPACK::LAPACK`?

Reply:
No.  That dependency is transitive.  Some lapack implementations have a built-in blas (OpenBLAS), and other implementations require 3 libraries (MKL).  Remember, the name here is a cmake-target, not a library.